maven/bnd-export-maven-plugin/src/main/java/aQute/bnd/maven/export/plugin/ExportMojo.java

```diff
@@ -9,27 +9,46 @@
 import org.apache.maven.plugin.AbstractMojo;
 import org.apache.maven.plugin.MojoExecutionException;
 import org.apache.maven.plugin.MojoFailureException;
+import org.apache.maven.plugins.annotations.Component;
 import org.apache.maven.plugins.annotations.Mojo;
 import org.apache.maven.plugins.annotations.Parameter;
 import org.apache.maven.plugins.annotations.ResolutionScope;
+import org.apache.maven.project.MavenProject;
+import org.apache.maven.project.ProjectDependenciesResolver;
+import org.eclipse.aether.RepositorySystem;
+import org.eclipse.aether.RepositorySystemSession;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import aQute.bnd.build.Workspace;
+import aQute.bnd.maven.lib.resolve.DependencyResolver;
 import aQute.bnd.osgi.Constants;
+import aQute.bnd.repository.fileset.FileSetRepository;
 import aQute.bnd.service.RepositoryPlugin;
 import biz.aQute.resolve.Bndrun;
 
 @Mojo(name = "export", defaultPhase = PACKAGE, requiresDependencyResolution = ResolutionScope.COMPILE_PLUS_RUNTIME)
 public class ExportMojo extends AbstractMojo {
 	private static final Logger	logger	= LoggerFactory.getLogger(ExportMojo.class);
 
+	@Parameter(defaultValue = "${project}", readonly = true, required = true)
+	private MavenProject				project;
+
+	@Parameter(defaultValue = "${repositorySystemSession}", readonly = true, required = true)
+	private RepositorySystemSession		repositorySession;
+
 	@Parameter(readonly = true, required = true)
 	private List<File>	bndruns;
 
 	@Parameter(defaultValue = "${project.build.directory}", readonly = true)
 	private File		targetDir;
 
+	@Parameter(readonly = true, required = false)
+	private List<File>					bundles;
```

Comment:
I guess with this configuration setup, all bndruns must share the same FileSetRepo?